src/calculation/ClauseResultsBuilder.ts

```diff
@@ -875,10 +878,19 @@ export function getSDEValues(measure: fhir4.Measure, statementResults: cql.State
       if (sde.criteria?.expression) {
         const expression = sde.criteria.expression;
         const result = statementResults[expression];
+        const usage = sde.usage?.[0]?.coding?.[0].code;
+        if (!isSupplementalDataUsage(usage)) {
+          throw new UnexpectedProperty(
+            'Expected sde usage code from the MeasureDataUsage valueset: https://terminology.hl7.org/3.1.0/ValueSet-measure-data-usage.html'
+          );
+        }
         results.push({
           name: sde.code?.text || expression,
           rawResult: result,
-          pretty: prettyResult(result)
+          pretty: prettyResult(result),
+          id: sde.id,
+          criteriaExpression: expression,
+          usage: usage as SupplementalDataUsage
```

Comment:
Because you threw an error above if the type guard is not satisfied, TypeScript knows that from then on, `usage` must be of type `SupplementalDataUsage`, making this cast unnecessary now

```suggestion
          usage
```